v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/V2FConstants.scala

```diff
@@ -0,0 +1,244 @@
+package org.broadinstitute.monster.etl.v2f
+
+/**
+  * constants for v2f that allow for transforms of the converted tsv to json
+  */
+sealed trait V2FConstants {
+
+  /**
+    * the name of the tsv table that is being converted to Json and transformed
+    */
+  def tableName: String
+
+  /**
+    * relative path to the tsv that will be converted to Json and transformed
+    */
+  def tsvPath: String
+
+  /**
+    * the names of json fields that should be converted from a json string to a json double
+    */
+  def fieldsToConvertToJsonDouble: List[String]
+
+  /**
+    * the names of json fields that should be converted from a json string to a json double
+    */
+  def fieldsToConvertToJsonInt: List[String]
+
+  /**
+    * the names of json fields that should be converted from a json string to a json boolean
+    */
+  def fieldsToConvertToJsonBoolean: List[String]
+
+  /**
+    * the names json fields that should be converted from from a json string to a json array
+    * the keys of the map are delimeter's and the values of map are names of json fields for the give delimeter(key)
+    */
+  def fieldsToConvertToJsonArray: Map[String, List[String]]
+
+  /**
+    * the json fields of a json array that should be converted from a json string to a json double
+    */
+  def fieldsToConvertFromJsonArrayStringToDouble: List[String]
+}
+
+case object FrequencyAnalysis extends V2FConstants {
+  override def tableName = "Frequency Analysis"
+
+  override def tsvPath: String = "frequencyanalysis/*/*"
+
+  override def fieldsToConvertToJsonDouble: List[String] = List(
+    "eaf",
+    "maf"
+  )
+
+  override def fieldsToConvertToJsonInt: List[String] = ???
```

Comment:
`???`s are a code smell. If these aren't exploding in the main pipeline, it means they aren't being called, so we must have hard-coded paths for each table type. I think that's OK, and we should scrap the top-level `sealed trait` to avoid needing to declare pointless overrides here.

Alternatively these could be filled in as `Nil`, and we could lift the common steps of the main pipeline to a generic method that takes a `V2FConstants` instance without knowing anything special about it.

Reply:
All of ^^^ aside, for this table the value should be `List("position")`